### PR TITLE
Fix .gitattributes

### PR DESCRIPTION
**Pull request by VincentLanglet**
Hi @Nyholm @nicolas-grekas, seems like some entries were missing in the gitattributes definition

### CODE REVIEW

**Comment by VincentLanglet** [comment]
I assume failures are unrelated

File: .gitattributes [hunk 1]
```diff
@@ -4,10 +4,11 @@ tests/                 export-ignore
 .editorconfig          export-ignore
 .gitattributes         export-ignore
 .gitignore             export-ignore
-.php_cs                export-ignore
+.php-cs-fixer.dist.php export-ignore
 .scrutinizer.yml       export-ignore
 .travis.yml            export-ignore
 phpstan.neon.dist      export-ignore
-phpstan.baseline.dist  export-ignore
+phpstan-baseline.dist  export-ignore
```
Comment (stof) [inline .gitattributes:11]: the extension is `.neon`, not `.dist

Comment (VincentLanglet) [reply .gitattributes]: Indeed, fixed.

**Review by stof — APPROVED** [review]

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**